coffee/chosen.jquery.coffee

```diff
@@ -510,3 +510,7 @@ class Chosen extends AbstractChosen
         w = f_width - 10
 
       @search_field.css({'width': w + 'px'})
+
+  form_field_has_changed: (extra) ->
```

Comment:
Also I'm not sure about the name of the method. More leaning to `trigger_form_field_change`, but maybe some of @harvesthq/chosen-developers can weigh in on this matter?